css/base.css

```diff
@@ -2,51 +2,70 @@
 
 body {
   width: 100%;
-  font-family: sans-serif;
-  line-height: 1.2em;
+  font-family: 'Open Sans Condensed', sans-serif;
```

Comment:
The styling for specific containers (body, header, etc.) would be good to put into the modules.css file, following SMACSS